qa/standalone/scrub/osd-scrub-test.sh

```diff
@@ -351,6 +351,7 @@ function _scrub_abort() {
     fi
 
     ceph osd set $stopscrub
+    ceph osd set noscrub
```

Comment:
I wish this was like the code above that does things conditionally:

``` 
     if [ "$type" = "deep_scrub" ];
      then
       ceph osd set noscrub
     fi
```

Reply:
OK w me. Will change and push.